pom.xml

```diff
@@ -0,0 +1,232 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+	This is a template for a Reactome Maven based project.
+	Please customize this template by filling in values in square brackets.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>org.reactome.[optionalsubdomain]</groupId>
+	<artifactId>[lower-case-name-using-kebab-case]</artifactId>
+	<version>1.0-SNAPSHOT</version>
```

Comment:
Should be 0.0.1 for a project that's just starting.

Reply:
Ah okay - I think the version number was auto-generated.  I'll change it.

Reply:
Done: https://github.com/reactome/maven-project-template/pull/1/commits/7168ce131205c64dde359f8178534940db07bb68